resources/candlepin-lint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+
+env | sort
+echo
+
+docker pull docker-registry.usersys.redhat.com/candlepin/candlepin-postgresql || true
+echo "Docker pull completed"
+
+# The docker container test script will know to copy out
+echo "Using workspace: $WORKSPACE"
+mkdir -p $WORKSPACE/artifacts/
+
+# make selinux happy via http://stackoverflow.com/a/24334000
+chcon -Rt svirt_sandbox_file_t $WORKSPACE//artifacts/
+
+# Fix PR author if necessary
+if [ "$ghprbPullAuthorEmail" == "" ]; then
+  `git log -n 1 --format="export ghprbPullAuthorEmail=%aE"`
+fi
+
+echo "Using PR author e-mail: $ghprbPullAuthorEmail"
```

Comment:
@Ceiu, if someone triggers the jobs outside of a PR (ex. manually runs the job in Jenkins), then we (plugin or no) have no way of knowing which PR it is tied to (ex. two PRs with same commit could have different author). We could move this logic to the pipeline or just remove it for now. 

Could probably do something like the following (groovy):

```groovy
destination = binding.variables.get('ghprbPullAuthorEmail') ?: binding.variables.get('GIT_AUTHOR_EMAIL') ?: 'chainsaw@redhat.com'
```

Thoughts?